cmd/influxd/recovery/auth/auth.go

```diff
@@ -133,7 +144,10 @@ func (cmd *authCreateCommand) run() error {
 	defer store.Close()
```

Comment:
Can this fail?

Reply:
Error is captured now.

Reply:
`store.Close()` error not captured, I think

Reply:
Ah, I captured the error from another `store.Close()`. Will be fixed in next commit.